go.mod

```diff
@@ -1,11 +1,16 @@
 module github.com/Microsoft/go-winio
 
-go 1.21
+go 1.22.0
```

Comment:
Did the updated modules enforce updating to `go1.22.0`, or does it accept staying on `go1.21` ?

Reply:
Yes, it needs go 1.22:
```
$ GOTOOLCHAIN=go1.21.0 go get golang.org/x/tools/cmd/stringer
go: golang.org/x/tools/cmd/stringer: golang.org/x/tools@v0.29.0 requires go >= 1.22.0 (running go 1.21.0; GOTOOLCHAIN=go1.21.0)
```